### ISSUE TITLE
[Feature] Supporting more formats like SWF?

### ISSUE THREAD

**Issue by ghost** [stashapp/stash#1611]
**Is your feature request related to a problem? Please describe.**
I am very new to github so i hope i didn't miss a post or info concerning this "issue" BUT i have many animations in my collection that are sadly in a SWF format and i would love to integrate them here.

**Describe the solution you'd like**
I am not a software engineer, so i don't know if it would be possible to integrate a player for those animations, but i would really appreciate if it were possible.

**Describe alternatives you've considered**
I already tried converting the animations to another supported format like MP4, but only with bad results because that barely works. Even for simple non-interactive animations. Right now i'm using another program, but yours if way better for that kind of stuff, so i would like to only have Stash for pics and animations/videos and the other one for games.

**Comment by stg-annon** [comment]
Flash files are mostly [dead at this point](https://community.adobe.com/t5/flash-player/viewing-swf-after-december-31st-2020/td-p/11733415) there are a few legacy standalone players that will work for this content but as for Stash I don't see them ever being supported.

Best case for you to get that content into stash is to convert the non interactive ones to mp4's with a tool that you find works best, as for interactive files your only option will most likely be a standalone flash player

**Comment by 642kradeht** [comment]
Yeah, thats what i thought. With it not being supported anymore i got myself flashpoint and i just hoped they may do it. But as you said, its a bit of a hassle to implement a thing thats no longer supported/used. Guess i will just take the program i already have them on ^^

**Comment by laurus-lx** [comment]
How about adding a button to have Stash launch video in an external player? 
- I found that Stash will transcode a lot of my wmv and avi files - resulting in a significant loss of quality and resolution. So what I do instead is get stream link from Stash and paste it in to VLC. This works, but somewhat awkward - having a button to launch a video in a new instance of VLC would be ideal

**Comment by feederbox826** [comment]
Closing as wontfix as it's also dependent on upstream ffmpeg and now libvips, and swf is the macro format, [`flv`](https://en.wikipedia.org/wiki/Flash_Video) would be more appropiate as it's just wrapped mp4
